src/imitation/policies/exploration_wrapper.py

```diff
@@ -53,9 +55,14 @@ def __init__(
         # Choose the initial policy at random
         self._switch()
 
-    def _random_policy(self, obs: np.ndarray) -> np.ndarray:
+    def _random_policy(
+        self,
+        obs: np.ndarray,
+        _state: Optional[Tuple[np.ndarray, ...]],
+        _episode_start: Optional[np.ndarray],
+    ) -> Tuple[np.ndarray, Optional[Tuple[np.ndarray, ...]]]:
```

Comment:
Nit: our style guide https://google.github.io/styleguide/pyguide.html#214-decision favors deleting variables to suppress unused variable warnings.